tests/integration/pipelines_integration_test.py

```diff
@@ -169,6 +169,26 @@ def test_import_pipeline_from_json_with_existing_jobs(create_global_project):
         )
         assert result
 
+    @staticmethod
+    def test_import_pipeline_from_json_without_alerting(create_global_project):
+        conf = create_global_project
+        job_name = "test_job_python"
+        jobs = conf.saagie_api.jobs.list_for_project_minimal(conf.project_id)["jobs"]
+        job = list(filter(lambda j: j["name"] == job_name, jobs))
+        print(job)
```

Comment:
remove print?

Reply:
you're right. I removed it (and also done on the test test_import_pipeline_from_json_with_existing_jobs)